core/datainput.gms

```diff
@@ -1415,6 +1415,20 @@ pm_emifac(ttot,regi,"seliqfos","fehos","tdfoshos","co2") = p_ef_dem(regi,"fehos"
 pm_emifac(ttot,regi,"seliqfos","fepet","tdfospet","co2") = p_ef_dem(regi,"fepet") / (sm_c_2_co2*1000*sm_EJ_2_TWa); !! GtC/TWa
 pm_emifac(ttot,regi,"seliqfos","fedie","tdfosdie","co2") = p_ef_dem(regi,"fedie") / (sm_c_2_co2*1000*sm_EJ_2_TWa); !! GtC/TWa
 
+*** define non-energy emission factors. For now only for the chemicals industry
+
+pm_emifacNonEnergy(ttot,regi,'segafos','fegas','indst','co2') = 0.051; !! GtC/TWa
+pm_emifacNonEnergy(ttot,regi,'seliqfos','fehos','indst','co2') = 0.069; !! GtC/TWa
+pm_emifacNonEnergy(ttot,regi,'sesofos','fesos','indst','co2') = 0.086; !! GtC/TWa
+
+pm_emifacNonEnergy("2005",regi,'segafos','fegas','indst','co2') = 0.068; !! GtC/TWa
```

Comment:
Hi Simon, 

I am not sure I understand why emission factors change over time? 
especially as they seem monotonously increasing in 2005 and 2010 - doesn't this create a problem when they revert down to the ttot value in 2015?

ahhhh - is this emission factor a "real emission factor" multiplied by a "share of FE oxidized in the process"? 
If yes, I think it would be much clearer to split out the two parameters, and have them as separate inputs - otherwise it will be very painful to debug and discuss with other modelers. 

in a similar vein, I think we usually try to enter values into REMIND (where possible) in units that make sense, not REMIND units. So I would rather go for a default unit like kg/TJ (as used by the IPCC) and then convert with the correct sm_c_XXX conversion factors to REMIND units 
Similarly, how can emission factors be larger than the

Reply:
The emission factors are based on the aggregated values provided by Silvia for Germany here:
https://mattermost.pik-potsdam.de/rd3/pl/phprpx66ntdufjyu196s6add8e
They are split by liquids, solids, gases using a crude approximation that assumes the carbon content of each energy carrier and that all of these energy carriers share the same efficiency in the chemical processes. This needs to be improved/validated by someone that knows more about how the chemicals industry work. 

The emission factors include the temporal dimension because they vary substantially from year to year. They can be calculated for the different years using the crf tables. 

I already mentioned this before, but the UNFCCC crf emission tables should be used to calculate these emission factor values per country directly in mrremind. They vary substantially from country to country as the chemicals industry has quite different focus in different countries. By doing this you can create the input data respecting the country differences, the temporal differences and be provided to REMIND in the relevant unit.

Reply:
> I already mentioned this before, but the UNFCCC crf emission tables should be used to calculate these emission factor values per country directly in mrremind.

If you don't have the CRF data in mrremind already, I've extracted the data which is available from the unfccc's API already: https://zenodo.org/record/5752337#.Yt7CRkhBzMU

Unfortunately, what is available from the API is not reflecting what is available in the submissions fully and is delayed (someone at the UNFCCC has to put it into their database first). There is an effort to read the data straight from the tables, coordinated here: https://gin.hemio.de/jguetschow/UNFCCC_non-AnnexI_data

A nicely curated data set is [primap-crf](https://zenodo.org/record/4723476#.Yt7D_0hBzMU), but it doesn't include emission factors (not even implicitly) because it doesn't include activity data.

Maybe you know all of this already, I just wanted to make sure that there is no duplication of work :sweat_smile:

Reply:
this was addressed by Falk, here: https://github.com/pik-piam/mrremind/pull/313
thank you!!